### DISCUSSION TITLE
Lost TalosConfig

### DISCUSSION THREAD

**Discussion by binaryn3xus** [siderolabs/talos#8675] [Q&A]
If someone loses their TalosConfig, can it be re-downloaded or generated from the running cluster?

I re-imaged my main workstation and realized that I didn't save my TalosConfig file. Can I get it back somehow? Or does it require a cluster destroy and redeploy?

**Comment by steverfrancis** [comment]
https://www.talos.dev/v1.7/talos-guides/howto/cert-management/#from-control-plane-machine-configuration

**Reply by narrator-z** [reply]
I also encountered the same problem. 
When deploying the patch, I accidentally overwrote the original one. I tried the methods in the link above but it didn't work.

**Reply by smira** [reply]
You need to clarify "it didn't work", as there's nothing I could help without details.
